golem-ts-sdk/tests/setup.ts

```diff
@@ -12,14 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { TypeMetadata } from '../src/typeMetadata';
+// Load type metadata loads the type-metadata that was saved into .metadata directory
+// when running `npm run test`
```

Comment:
Test case setup, here we load the type metadata from .metadata directory.
This code is moved to golem-ts-types-core library in the next version, so that it is just `TypeMetadata.loadFromMetadataDir()`. Unfortunately I placed that functionality in `typegen` library, which the SDK shouldn't depend on